docs/arch/adr-25.md

```diff
@@ -0,0 +1,161 @@
+          {"value": 1},
+          {"value": [
+          {"option-3": 1},
+          {"option-4": 1},
+          {"option-5": 1}
+        ]}
+      ]
+    }
+  ]}
+]}
+```
+
+The annotation structure for the sub-task, under `details`, has a few issues because it solely relies on an array index to relate back to the original sub-task. This makes it difficult to make downstream analysis and aggregation scripts. The aggregation code now has to parse the details array and make a "mock annotation" of the correct structure to be passed along to the next reducer.
+
+### Sub-task UI
+
+The sub-task UI positioned itself fixed below relative to the position of the mark. Notably transcription projects have commented that this interferes with being able to transcribe successfully since the dialog may cover up part of the subject and cannot be moved without moving the drawing mark.
+
+## Decision
+
+For initial support, we will support the single and multiple choice question tasks and the text task in the sub-task. Slider task may be deprecated and dropdown task may be changing in ways we do not have a plan for yet, so they can be supported later if it makes sense to add them.
+
+### Annotation JSON structure
+
+The annotations in the details array will be updated to be an object that just contains a reference to the sub-task's unique identifier. The task annotation itself will be stored in the classification's annotations array flattened. 
+
+The main benefit of this reorganization will be with downstream analysis and aggregation. When aggregating drawn shapes the first step is clustering. Once the clusters are found the subtasks need to be aggregated within each cluster. This will be easier to do if the structure of each subtask annotation is the same as if that task was asked on its own. The code can just take all subtask annotations within a cluster and just pass it to the reducer as if it is a list of main task annotations without having to reshape them.
+
+An addition of `markIndex` is being added for the sub-task annotations for the purpose of having an identifier relating it back to the parent drawing task annotation value array which represents marks. 
+
+An example of the new sub-task annotation JSON structure at classification submission:
+
+```json
+{"annotations": [
+  {
+    "task": "T0",
+    "taskType": "drawing",
+    "value": [
+      {
+        "frame": 0,
+        "tool": 0,
+        "toolType": "point",
+        "x": 452.18341064453125,
+        "y": 202.87478637695312,
+        "details": [
+          {"task": "T0.0.0"},
+          {"task": "T0.0.1"}
+        ]
+      },
+      {
+        "frame": 0,
+        "tool": 1,
+        "toolType": "point",
+        "x": 404.61279296875,
+        "y": 583.4398803710938,
+        "details": [
+          {"task": "T0.1.0"},
+          {"task": "T0.1.1"}
+        ]
+      }
+    ]
+  },
+  {
+    "task": "T0.0.0",
+    "taskType": "single",
+    "markIndex": 0,
+    "value": 0
+  },
```

Comment:
Maybe add this to the example to show how two marks from the same tool would be represented?
```json
{
    "task": "T0.0.0",
    "taskType": "single",
    "markIndex": 2,
    "value": 0
}
```

Reply:
I've included this. Thanks.